lib/order.rb

```diff
@@ -0,0 +1,100 @@
+require "csv"
+require_relative "customer"
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status, :total
```

Comment:
You define a reader called `total` here, but redefine it below at line 20. If you're only ever going to use the `total` at line 20, done use an `attr_reader` here.